dtm0/service.c

```diff
@@ -62,9 +62,10 @@ static const struct m0_reqh_service_type_ops dtm0_service_type_ops = {
 };
 
 static const struct m0_reqh_service_ops dtm0_service_ops = {
-	.rso_start = dtm0_service_start,
-	.rso_stop  = dtm0_service_stop,
-	.rso_fini  = dtm0_service_fini
+	.rso_start           = dtm0_service_start,
+	.rso_stop            = dtm0_service_stop,
+	.rso_fini            = dtm0_service_fini,
+	.rso_prepare_to_stop = dtm0_service__ha_unsubscribe,
```

Comment:
Propose to name consistently, dtm0_service_prepare_to_stop() which calls dtm0_service__ha_unsubscribe(). Possibly dtm0_service_prepare_to_stop() will be extended.

Reply:
makes sense, I'll change that.